internal/output/html/package_table_template.gohtml

```diff
@@ -55,19 +55,48 @@
   <tr>
     <td colspan="100%">
       <div id="table-tr-{{ $index }}-details" class="package-details">
-        {{ if and $element.LayerDetail (not (eq $element.LayerDetail.LayerID "")) }}
+        {{ if and $element.LayerDetail.LayerInfo.LayerMetadata (not (eq $element.LayerDetail.LayerInfo.LayerMetadata.Command "")) }}
+        {{ $index := $element.LayerDetail.LayerIndex }}
+        {{ $diffID := $element.LayerDetail.LayerInfo.LayerMetadata.DiffID }}
+        {{ $baseImageIndex := $element.LayerDetail.LayerInfo.LayerMetadata.BaseImageIndex }}
+        {{ $originalCommand := $element.LayerDetail.LayerInfo.LayerMetadata.Command }}
+        {{ $commandSet := formatLayerCommand $originalCommand }}
+        {{ $command := index $commandSet 0 }}
+        {{ $commandDetail := index $commandSet 1 }}
+        {{ $longCommand := false }}
+        {{ if gt (len $command) 109 }}
+        {{ $longCommand = true }}
+        {{ end }}
+        {{ $binaries := formatSlice $element.OSPackageNames }}
+        {{ if not (eq $binaries "") }}
+        <p><span class="package-detail-title">Installed binaries:</span> {{ $binaries }}</p>
+        {{ end }}
+        <div class="layer-command-container">
+          <p><span class="package-detail-title">Layer introduced in:&nbsp;</span></p>
+          <div class="tooltip">
+            <p># {{ $index }} layer</p>
+            <span class="tooltiptext">Diff ID: {{ $diffID }}</span>
```

Comment:
nit: can we show the command of the layedr instead of the diffID? diffID isn't very useful for a human to look at.

Reply:
It does show the layer command here. The diff ID is hidden and only be visible when a user hovers the Layer index.
![Screenshot 2025-01-21 at 2 17 01 PM](https://github.com/user-attachments/assets/0861b3b6-7ff7-4abc-8e2c-c1d81f7525aa)

Reply:
Ah missed that, thanks!